src/context/directory/index.ts

```diff
@@ -24,7 +25,7 @@ export default class DirectoryContext {
     this.filePath = config.AUTH0_INPUT_FILE;
     this.config = config;
     this.mappings = config.AUTH0_KEYWORD_REPLACE_MAPPINGS || {};
-    this.mgmtClient = mgmtClient;
+    this.mgmtClient = pagedClient(mgmtClient);
```

Comment:
Bit of a blunder that the management client for the directory and YAML contexts do not have pagination enabled. Digging in, there is some technical reason why this is, but at face value is odd and made this less trivial than it should have been.